### DISCUSSION TITLE
Kodi на Xbox

### DISCUSSION THREAD

**Discussion by korn3r** [quarckster/kodi.kino.pub#233] [Q&A] [answered]
Здравствуйте.
Установил аддон в коди на xbox (sx).
Любой переход по меню аддона занимает ~10 секунд, в том числе поиск и кнопка "профиль".
Сами фильмы при этом открываются моментально, перемотка работает.
Веб-версия сайта в бразуере на том же самом устройстве грузится моментально.
Подскажите пожалуйста куда можно попробовать покопать, хочется именно коди, а не браузер.

**Comment by quarckster** [comment, marked as answer]
Здравствуйте, не думаю, что в этой ситуации что-то можно сделать. "Поиск", "Профиль" и практически все остальные действия плагина работают через API кинопаба. Также Kodi не самый оптимизированный софт, а система плагинов реализована на Python, что тоже накладывает отпечаток.

**Reply by korn3r** [reply]
Спасибо за ответ. 
На форуме коди уже нагуглил что проблема после какого-то апдейта на бокс возникла.
https://forum.kodi.tv/showthread.php?tid=369438

Говорят даже пофикшенный коди каким-то способом можно воткнуть на бокс. Надеюсь в официальном релизе пофиксят.
Зря я на плагин кинопаба подумал сначала :)
